src/controllers/users.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const { UserModel } = require('../models/users');
+
+const getAllUsers = (req, res) => {
+  const users = UserModel.getUsers();
+
+  res.send(users);
+};
+
+const addNewUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.status(400).send('Bad request');
+
+    return;
+  }
+
+  const newUser = UserModel.addUser({ name });
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const getOneUser = (req, res) => {
+  const { userId } = req.params;
+  const user = UserModel.getUser(+userId);
+
+  if (!user) {
+    res.status(404).send('User not found');
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const deleteUser = (req, res) => {
+  const { userId } = req.params;
+
+  const user = UserModel.removeUser(+userId);
+
+  if (!user) {
+    res.status(404).send('User not found');
+
+    return;
+  }
+
```

Comment:
HTTP status code 204 (No Content) should not include a message body. Consider using `res.status(204).send()` without any arguments.